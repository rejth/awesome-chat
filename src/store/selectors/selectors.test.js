import { describe, test, expect } from '@jest/globals';
import { getChatData, getAllChannels } from './index.js';

describe('Test selectors', () => {
  test('getChatData selector', () => {
    expect(getChatData({})).toEqual({});
    expect(getChatData({})).toMatchSnapshot();
  });
  test('getAllChannels selector', () => {
    expect(getAllChannels({})).toEqual([]);
    expect(getAllChannels({})).toMatchSnapshot();
  });
});
