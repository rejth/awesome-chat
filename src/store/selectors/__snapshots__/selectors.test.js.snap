// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test selectors getAllChannels selector 1`] = `Array []`;

exports[`Test selectors getChatData selector 1`] = `Object {}`;
